jbcli/jbcli/cli/jb.py

```diff
@@ -553,6 +552,55 @@ def prompt_interval():
     stash.put("interval", answer)
 
 
+def prompt_name():
+    question = [
+        {
+            "type": "input",
+            "name": "name",
+            "message": "Enter First and Last Name separted by space",
+        }
+    ]
+    answer = prompt(question)["name"]
+    name_parts = answer.split(' ')
+    stash.put("firstname", name_parts[0])
+    stash.put("lastname", name_parts[1])
+
+
+def prompt_email():
+    question = [
+        {
+            "type": "input",
+            "name": "email",
+            "message": "Enter email address to be created",
+        }
+    ]
+    answer = prompt(question)["email"]
+    stash.put("email", answer)
+
+
+def check_user_email():
+    print("Checking that user email exists locally")
+    try:
+        user_email = stash.get('email')
+        if user_email is None:
+            echo_warning("User email not found locally")
+            prompt_email()
+    except Exception as e:
+        pass
```

Comment:
why swallow errors here (and in check_user_name)? I don't see anything that will obviously fail in the common case, so I think it's better to just let any errors bubble up (or catch a specific exception there's one we care about in particular)